client/src/main.rs

```diff
@@ -154,6 +154,14 @@ pub struct Opt {
     /// How long to wait before the auto flush check fires
     #[clap(long, global = true)]
     flush_timeout: Option<u32>,
+
+    /// IP:Port for the oximeter register address
+    #[clap(long, global = true, default_value = "127.0.0.1:12221")]
+    oximeter_register: SocketAddr,
+
+    /// IP:Port for the oximeter listen address
```

Comment:
Also kind of a nit, but I'd call this `metric_collect_address` or something like that. This is where this process will listen for metric collection requests from `oximeter`.

Reply:
I'll update that.  Maybe "metric_register" and "metric_collect" as names for these?  Or do you feel partial to the "metric_collect_address" full name vs. the shorter one?  I sometimes make things too terse.